A new makefile to do the installation o the pre-existing binaries and
the pkg-config file
--- Makefile	1970-01-01 01:00:00.000000000 +0100
+++ Makefile	2018-09-25 20:42:08.686722661 +0100
@@ -0,0 +1,28 @@
+all:	libefwfilter.pc
+
+%.pc:	%.pc.in
+	sed -e "s!@LIBDIR@!$(LIBDIR)!g" \
+		-e "s!@VERSION@!$(VERSION)!g" < $< > $@
+
+install:
+	mkdir -p $(LIBDIR) $(INCDIR) $(PKGCONFDIR) $(UDEVDIR)
+	case $(DEB_BUILD_ARCH) in \
+	i386) \
+		cp $(SRCDIR)/lib/x86/libEFWFilter.a $(LIBDIR); \
+		cp $(SRCDIR)/lib/x86/libEFWFilter*.so.$(VERSION) $(LIBDIR);; \
+	amd64) \
+		cp $(SRCDIR)/lib/x64/libEFWFilter.a $(LIBDIR); \
+		cp $(SRCDIR)/lib/x64/libEFWFilter*.so.$(VERSION) $(LIBDIR);; \
+	arm64) \
+		cp $(SRCDIR)/lib/armv8/libEFWFilter.a $(LIBDIR); \
+		cp $(SRCDIR)/lib/armv8/libEFWFilter*.so.$(VERSION) $(LIBDIR);; \
+	armhf) \
+		arch=`uname -m | sed -e 's/[lb]$$//'`; \
+		cp $(SRCDIR)/lib/$$arch/libEFWFilter.a $(LIBDIR); \
+		cp $(SRCDIR)/lib/$$arch/libEFWFilter*.so.$(VERSION) $(LIBDIR);; \
+	esac
+	ln -sf libEFWFilter.so.$(VERSION) $(LIBDIR)/libEFWFilter.so.$(MAJOR_REV)
+	ln -sf libEFWFilter.so.$(VERSION) $(LIBDIR)/libEFWFilter.so
+	cp $(SRCDIR)/include/*.h $(INCDIR)
+	cp $(SRCDIR)/libefwfilter.pc $(PKGCONFDIR)
+	cp $(SRCDIR)/70-asi-fw.rules $(UDEVDIR)
